kmod/src/lock.c

```diff
@@ -1475,7 +1490,10 @@ void scoutfs_free_unused_locks(struct super_block *sb)
 		.nr_to_scan = INT_MAX,
 	};
 
-	linfo->shrinker.shrink(&linfo->shrinker, &sc);
+	// shouldn't need to KC_INIT_SHRINKER_FUNCS as we're calling the shrinker
```

Comment:
(Don't use // comments in the kernel.)   And also just remove this comment, it's fine to just make the direct call without commentary.